.github/CONTRIBUTING-en-US.md

```diff
@@ -1,42 +1,97 @@
 
-If you have not yet contributed to an open source project or are not familiar with this project, there are tasks for people like you in the [Issues](https://github.com/okfn-brasil/querido-diario-comunidade/issues) tab, labeled as [`good first issues`](https://github.com/okfn-brasil/querido-diario-comunidade/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue").
+Knowing `git` and how the *GitHub* platform works is a crucial step. We recommend this [Digital Ocean tutorial](https://www.digitalocean.com/community/tutorials/how-to-use-git-effectively) if you are not familiar with them.
+
+### Figuring out what to contribute with
 
-If any of those interest you, before starting, you can leave comments to ask any relevant questions and then leave a notice that you will be working on that task.
+#### Issues
 
-We also have a [discussion forum](https://github.com/okfn-brasil/querido-diario-comunidade/discussions) where you can contribute ideas related to the project.
+On the **Issues** tab of the chosen repository, there are tasks or issues
+specific to that repository. Check to see if any of them interest you.
+- When a task is suitable for beginners, it will be labeled as `good first issue`.
+- When you find a task that looks interesting to you, comment that you will do that task. That way, other folks will know that you're already working on it, and can offer to help you or take on other tasks that haven't been worked on yet.
+- Whether or not you've already chosen an *issue* to work on, you can also comment on relevant questions regarding the task.
 
-And a [community on Discord](https://bit.ly/discord-ok), if you want to ask questions faster or talk to people who are also interested in Open Knowledge Brasil's civic innovation projects.
+#### Milestones
+
+If there are no open *issues* or none interest you, look at the **Milestones** of the repository you want to contribute to. These indicate the goals to be achieved. Exploring the project files looking for areas for improvement that would help the repository reach a *milestone* is an excellent way of contributing.
+- When you find something, open an *issue* describing the potential improvement. If you are also planning to work on that task, add that to the issue.
+
+#### Discussions
+
+In case you have an idea or suggestion for the project's direction, open a **Discussion**.
+- As *Querido Diário* is made up of many repositories and discussions tend to be transversal, it is **a management's decision** to keep them all centralized. So, to start or follow discussions, just look at the tab [`Discussion` in `querido-diario-community`](https://github.com/okfn-brasil/querido-diario-comunidade/discussions).
+
+#### Community chat
+
+Join our [community](https://go.ok.org.br/discord). There, you'll be able to ask
+quick questions, interact with the project maintainers, learn about how can you
+help and chat with other *Querido Diário* contributors.
+
+### Interacting with reviewers
```

Comment:
Já que no geral foi retirada a ênfase em revisões de código no restante to texto, tomei a liberdade de retirar daqui também